Libplanet/Crypto/PrivateKey.cs

```diff
@@ -269,6 +265,19 @@ private static ECPrivateKeyParameters GenerateKeyParam()
             return gen.GenerateKeyPair().Private as ECPrivateKeyParameters;
         }
 
+        private static ECPrivateKeyParameters GenerateKeyFromBytes(byte[] privateKey)
+        {
+            var param = new ECPrivateKeyParameters(
+                "ECDSA",
+                new BigInteger(1, privateKey),
+                GetECParameters()
+            );
+
+            var key = new PrivateKey(param);
+            var publicKey = key.PublicKey;
```

Comment:
This statement also seems unnatural, so it would be better if there is some comment about it.